decidim-core/lib/decidim/content_renderers/hashtag_renderer.rb

```diff
@@ -29,8 +29,9 @@ class HashtagRenderer < BaseRenderer
       # @return [String] the content ready to display (contains HTML)
       def render(links: true, extras: true, html_escape: false, strip_tags: false)
         text = content
-        text = strip_tags(text) if strip_tags
+        text = sanitize(text, tags: %w(strong em a)) if strip_tags
```

Comment:
How are we going to add the `target` and `rel` to regular links? Why add `strong` and `em`?